src/Mod/CAM/Path/Tool/machine/ui/machine.py

```diff
@@ -0,0 +1,399 @@
+
+        general_group.setLayout(general_layout)
+        self.layout.addWidget(general_group)
+
+        # Axis Properties group
+        axis_properties_group = QtGui.QGroupBox(translate("CAM", "Axis Properties"))
+        axis_properties_layout = QtGui.QGridLayout()
+        self.axis_edits = {}
+
+        # Add column headers for the grid
+        axis_properties_layout.addWidget(QtGui.QLabel(translate("CAM", "Min")), 0, 1)
+        axis_properties_layout.addWidget(QtGui.QLabel(translate("CAM", "Max")), 0, 2)
+        axis_properties_layout.addWidget(QtGui.QLabel(translate("CAM", "Rigidity")), 0, 3)
+        axis_properties_layout.addWidget(QtGui.QLabel(translate("CAM", "Max Feed")), 0, 4)
+
+        # Linear Axes
+        for i, (axis_name, axis) in enumerate(sorted(machine.axes.items())):
+            if not isinstance(axis, LinearAxis):
+                continue
+            row = i + 1  # Start from row 1 after headers
+
+            axis_properties_layout.addWidget(
+                QtGui.QLabel(f"{axis_name.capitalize()}-Axis:"), row, 0
+            )
+
+            start_edit = ui.createWidget("Gui::QuantitySpinBox")
+            start_edit.setProperty("value", axis.start)
+            axis_properties_layout.addWidget(start_edit, row, 1)
+
+            end_edit = ui.createWidget("Gui::QuantitySpinBox")
+            end_edit.setProperty("unit", "mm")
+            end_edit.setProperty("value", axis.end or FreeCAD.Units.Quantity("0 mm"))
+            axis_properties_layout.addWidget(end_edit, row, 2)
+
+            # Use FreeCAD.Units.Quantity does not support distance/N, so
+            # we use QDoubleSpinBox for rigidity to avoid unit issues
+            rigidity_edit = QtGui.QDoubleSpinBox()
+            rigidity_edit.setDecimals(6)
+            rigidity_edit.setMinimum(0.0)
+            rigidity_edit.setValue(axis.rigidity.Value)
+            rigidity_edit.setSuffix(" mm/N")
+            axis_properties_layout.addWidget(rigidity_edit, row, 3)
+
+            max_feed_edit = ui.createWidget("Gui::QuantitySpinBox")
+            max_feed_edit.setProperty("value", axis.max_feed)
+            axis_properties_layout.addWidget(max_feed_edit, row, 4)
+
+            self.axis_edits[axis_name] = (
+                start_edit,
+                end_edit,
+                rigidity_edit,
+                max_feed_edit,
+            )
+
+        # Angular axis
+        # Determine the starting row for angular axes based on the number of linear axes
+        angular_axis_start_row = (
+            len([a for a in machine.axes.values() if isinstance(a, LinearAxis)]) + 1
+        )
+        for i, (axis_name, axis) in enumerate(sorted(machine.axes.items())):
+            if not isinstance(axis, AngularAxis):
+                continue
+            row = angular_axis_start_row + i
+
+            axis_properties_layout.addWidget(
+                QtGui.QLabel(f"{axis_name.capitalize()}-Axis:"), row, 0
```

Comment:
```suggestion
                QtGui.QLabel(f"{axis_name.capitalize()}-axis"), row, 0
```
are these translatable?